src/messages/store/publish.ts

```diff
@@ -68,5 +68,5 @@ export async function Publish(spc: StorePublishConfiguration): Promise<string> {
         APIServer: spc.APIServer,
     });
 
-    return hash;
+    return message;
```

Comment:
You have to return the hash so that users can retrieve the content they pushed.

Reply:
Fixed in https://github.com/aleph-im/aleph-sdk-ts/pull/3/commits/b34344111a7430ecff5164032b9351736a298a7d